### PR TITLE
Add SerGo validator info

### PR DESCRIPTION
**Pull request by CrazySerGo**

### CODE REVIEW

File: src/config/validators/index.ts [hunk 1]
```diff
@@ -28,4 +28,17 @@ export const VALIDATOR_COMMUNITY = [
       ],
```
Comment (shamilkhan) [inline src/config/validators/index.ts:30]: @CrazySerGo Thanks for your PR! 
Please, add import of your thumbnail 
```ts
import { ReactComponent as SerGo } from './thumbnails/SerGo.svg';
```

File: src/config/validators/index.ts [hunk 2]
```diff
@@ -28,4 +28,17 @@ export const VALIDATOR_COMMUNITY = [
...
   },
+  {
+    name: 'SerGo',
+    Thumbnail: SerGo,
+    bio: `We validate with expertise, you earn with confidence.`,
+    email: 'contact@sergo.dev',
+    website: 'https://sergo.dev',
+    twitter: '',
+    validators: {
+      cere: [
```
Comment (shamilkhan) [inline src/config/validators/index.ts:41]: Please, run `yarn lint` in order to fix prettier issue
```
  Line 40:14:  Replace `⏎········'6SpjH8swCtFwmntQmdikMWyDKgr59q1cLauePYc2iqwwe6Bv',⏎······` with `'6SpjH8swCtFwmntQmdikMWyDKgr59q1cLauePYc2iqwwe6Bv'`  prettier/prettier
  ```

**Review by shamilkhan — CHANGES_REQUESTED** [review]

**Review by calmrat — DISMISSED** [review]
Identity info looks good.

**Comment by calmrat** [comment]
@CrazySerGo you'll need to merge conflicts on your branch; try `git pull` to make sure you're working with the latest upstream head; then resolve the merge any conflicts which are (eg) introduced by other community validator's updating this file at the time time.

**Comment by CrazySerGo** [comment]
@calmrat my PR was created 4 days ago but you decided to merge PR created a day ago that cause issue.
In my view it's a bad idea to modify one file by multiple validators if you don't care about queue.

**Comment by shamilkhan** [comment]
> @calmrat my PR was created 4 days ago but you decided to merge PR created a day ago that cause issue. In my view it's a bad idea to modify one file by multiple validators if you don't care about queue.

@CrazySerGo  Sorry, for that. 
This pull-request was ready faster I mean in terms of running checks. You still need to pull the latest changes from dev-cere and resolve conflicts and your Pull-Request will be merged.
Screenshot 2023-10-10 at 18 05 41

**Comment by CrazySerGo** [comment]
> > @calmrat my PR was created 4 days ago but you decided to merge PR created a day ago that cause issue. In my view it's a bad idea to modify one file by multiple validators if you don't care about queue.
> 
> @CrazySerGo Sorry, for that. This pull-request was ready faster I mean in terms of running checks. You still need to pull the latest changes from dev-cere and resolve conflicts and your Pull-Request will be merged. Screenshot 2023-10-10 at 18 05 41

Sorry, but I do not understand what you want.
Should I modify my request every time when another person creates a request?
I used updated file base on Xameyz merged request, so used latest version in my view. Where is conflict?
Unfortunately I'm not expert with github and <<<<dev-cere means nothing for me.

**Comment by shamilkhan** [comment]
> > > @calmrat my PR was created 4 days ago but you decided to merge PR created a day ago that cause issue. In my view it's a bad idea to modify one file by multiple validators if you don't care about queue.
> > 
> > 
> > @CrazySerGo Sorry, for that. This pull-request was ready faster I mean in terms of running checks. You still need to pull the latest changes from dev-cere and resolve conflicts and your Pull-Request will be merged. Screenshot 2023-10-10 at 18 05 41
> 
> Sorry, but I do not understand what you want. Should I modify my request every time when another person creates a request? I used updated file base on Xameyz merged request, so used latest version in my view. Where is conflict? Unfortunately I'm not expert with github and <<<<dev-cere means nothing for me.

Ok, let me create PR for you and resolve all the conflicts.

**Comment by shamilkhan** [comment]
@CrazySerGo I've added your changes in this [PR](https://github.com/Cerebellum-Network/staking-dashboard/pull/26). Please confirm that everything is OK.

**Comment by CrazySerGo** [comment]
> @CrazySerGo I've added your changes in this [PR](https://github.com/Cerebellum-Network/staking-dashboard/pull/26). Please confirm that everything is OK.

Thank you. Data is correct.

**Comment by shamilkhan** [comment]
@CrazySerGo Thank you! I've merged this [PR](https://github.com/Cerebellum-Network/staking-dashboard/pull/26) so we can close this one.

**Pull request closed without merging**